pkgs/development/libraries/libtorrent-rasterbar/1.2/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, lib, fetchFromGitHub, pkgconfig, automake, autoconf
+, zlib, boost, openssl, libtool, python, libiconv, geoip, ncurses
+}:
+
+let
+  version = "1.2.3";
+  formattedVersion = lib.replaceChars ["."] ["_"] version;
+
+  # Make sure we override python, so the correct version is chosen
+  # for the bindings, if overridden
+  boostPython = boost.override { enablePython = true; inherit python; };
+
+in stdenv.mkDerivation {
+  pname = "libtorrent-rasterbar";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "arvidn";
+    repo = "libtorrent";
+    rev = "libtorrent-${formattedVersion}";
+    sha256 = "0m1cvng7ihxy8p0d563kv4s4bqqzx1066zl4124zfnznsnxww0bf";
+  };
+
+  enableParallelBuilding = true;
+  nativeBuildInputs = [ automake autoconf libtool pkgconfig ];
+  buildInputs = [ boostPython openssl zlib python libiconv geoip ncurses ];
```

Comment:
`geoip` has been removed from libtorrent-rasterbar since 1.1: https://github.com/arvidn/libtorrent/blob/55460a60bc722f27d1300bddd9743d4d59492da7/include/libtorrent/session_handle.hpp#L550

Why the uses of `ncurses`? I can't find any reference to it in their repo?

Reply:
ncurses is required for the configure checks to succeed when building with python3. I have removed geoip from both versions and confirmed in the output binaries that nothing changed.